ui/app/router.js

```diff
@@ -192,12 +192,11 @@ Router.map(function () {
         });
       });
       this.route('policies', { path: '/policies/:type' }, function () {
-        this.route('index', { path: '/' });
         this.route('create');
-      });
-      this.route('policy', { path: '/policy/:type' }, function () {
-        this.route('show', { path: '/:policy_name' });
-        this.route('edit', { path: '/:policy_name/edit' });
+        this.route('policy', { path: '/:policy_name' }, function () {
```

Comment:
nesting `policy` inside `policies` follows the routing pattern we've been using. But I'm not sure if I'm missing something as to why this was done the other way. 

My guess was it had to do with policies having types? But I'm not sure 🤔